Tasks/day.js

```diff
@@ -1,15 +1,11 @@
-// Get day number
+'use strict';
 
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const Days = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```

Comment:
```suggestion
const DAYS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```
The Days is a global constant and must be written in caps.